kubernetes/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ set(SRCS
     src/apiKey.c
     src/apiClient.c
     src/binary.c
+    src/generic.c
     external/cJSON.c
     model/object.c
     model/admissionregistration_v1_service_reference.c
```

Comment:
Maybe kubernetes/include/generic.h is needed to added to 

```
set(HDRS
    include/generic.h
```
in kubernetes/CMakeLists.txt ?

Reply:
done.